### DISCUSSION TITLE
Suggestion: Indicate daylight savings time more prominently

### DISCUSSION THREAD

**Discussion by CyberFlameGO** [community/community#34192] [Other Feature Feedback, Questions, & Ideas]
In https://github.com/settings/profile, the list of timezones should reflect whether a zone is observing daylight saving or not, and a user's profile should indicate whether their timezone is observing daylight savings (to be more clear for a profile viewer).

**Comment by rcomer** [comment]
I just selected `(GMT + 00:00) London`, and it correctly shows on my profile as `UTC + 01:00` as we are currently in daylight savings time.  So the display seems right, but I agree the drop-down menu in the settings is a little ambiguous.

**Reply by CyberFlameGO** [reply]
Yeah exactly my point haha! I'd love it if the ambiguity was eliminated by specificity
